yarn.lock

```diff
@@ -408,6 +408,10 @@ buffer-shims@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/buffer-shims/-/buffer-shims-1.0.0.tgz#9978ce317388c649ad8793028c3477ef044a8b51"
 
+buffer-v6-polyfill@^1.0.3:
```

Comment:
Why is `buffer-v6-polyfill` necessary if you change `Buffer.from` to `new Buffer`?